Filter/Basic.php

```diff
@@ -203,13 +203,13 @@ public function getParameters()
 
     /**
      * Get the stream resource being filtered.
-     * Maybe available only during filter calls when the closing parameter is
-     * set to false.
+     * Maybe available only during `static::filter` calls when the closing
+     * parameter is set to false.
      *
      * @return  resource
      */
     public function getStream()
     {
-        return $this->stream;
+        return isset($this->stream) ? $this->stream : null;
```

Comment:
Property isn't documented (http://php.net/manual/en/class.php-user-filter.php),
it's possible but I just typed the following command:
```sh
$ php -r "print_r(new php_user_filter());"
php_user_filter Object
(
    [filtername] =>
    [params] =>
)
```

Do you have a real use case ?